vercel-build.sh

```diff
@@ -0,0 +1,4 @@
+rm -rf build html docs/source/autoapi
+cd docs
+make html
+cp -r build/html/ ../static/
```

Comment:
You need to cleanup all generated files before `make` is ran, and create `static`. Also add `/static` and `/docs/build/ to gitignore or delete build after it's ran